string labelName
int32 labelValue
bool isValid
